docs/Installation.md

```diff
@@ -97,11 +97,23 @@ sudo install -v -m 755 ./zm-testagent.lsb /etc/init.d/zm-testagent
 sudo install -v -m 755 ./tmpfiles.conf /usr/lib/tmpfiles.d/zonemaster.conf
 ```
 
+> If this is an update of Zonemster-Backend, you should remove any
+> `/etc/init.d/zm-backend.sh` and `/etc/init.d/zm-centos.sh` (scripts from
+> previous version of Zonemaster-Backend).
+
 ### 3.2 Database engine installation and configuration (CentOS)
 
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
+The installation instructions below assums that this is a new installation.

```

Comment:
"assums" -> "assumes"

There are more of these.

Reply:
Fixed * 3.